src/main/java/frc/robot/subsystems/drive/Gearbox.java

```diff
@@ -18,7 +18,7 @@ public Gearbox(Encoder encoder, CANSparkMax... controllers) {
         // TODO: these values are for a specific robot and specefic encoder
         // that may not be the same for all robots. Change them to reference
         // a value in Constants.java - Egan
-        this.encoder.setDistancePerPulse((6.0 * Math.PI) / 2048.0);
+        this.encoder.setDistancePerPulse((0.1524 * Math.PI) / 2048.0);
```

Comment:
Let's move this `0.1524` number to `DriveConstants.java` and call the variable `WHEEL_DIAMETER_METERS`

Reply:
Also get rid of the comment from Egan after doing that :)

Reply:
This thread was resolved but the change wasn't pushed up. Unresolving